base/array.jl

```diff
@@ -255,14 +255,14 @@ else
 end
 _default_eltype{I,T}(::Type{Generator{I,Type{T}}}) = T
 
-_array_for(T, itr, ::HasLength) = Array(T, Int(length(itr)::Integer))
-_array_for(T, itr, ::HasShape) = Array(T, convert(Dims,size(itr)))
+_array_for(T, itr, ::HasLength) = Array{T,ndims(itr)}(Int(length(itr)::Integer))
+_array_for(T, itr, ::HasShape) = Array{T,ndims(itr)}(convert(Dims,size(itr)))
 
```

Comment:
Oh, I see. I changed it to `Array{Int}` per your suggestion and so far the CI hasn't failed. Thanks for your help! 👍